modular_skyrat/modules/pixel_shift/code/pixel_shift.dm

```diff
@@ -57,3 +70,12 @@
 			if(pixel_x >= -16 + base_pixel_x)
 				pixel_x--
 				is_shifted = TRUE
+	if(abs(pixel_x > 8))
+		is_out_of_the_way = TRUE
+	else if(abs(pixel_y > 8))
+		is_out_of_the_way = TRUE
+	else
+		is_out_of_the_way = FALSE
+
+
+	density = is_out_of_the_way == FALSE // Inverted, cause we want it set to 0 if we *are* out of the way!
```

Comment:
i'm against fucking directly with the density on this, is there not a better way we can do this?

Reply:
Already doing it another way, don't worry!